### ISSUE TITLE
localfoodcoop.org domain?

### ISSUE THREAD

**Issue by ayoungethan** [openfoodsource/OFS_federation#17]
We have lingering references to the localfoodcoop.org domain, yet it seems to be defunct.  I wonder whether it makes sense to acquire the domain and forward it to openfoodsource.org, at least for a few years?

**Comment by guising** [comment]
AI agree. This makes sense but the domain is still registered to Bob Waldrop of the OKFC. I doubt he would be willing to relinquish it. If he would, this could also be used to host copies of the old software -- for archival/historical purposes, should anyone be interested.

Let's wait on this until OFS is more well-established and has a better draw.

**Comment by guising** [comment]
WHOIS localfoodcoop.org:
Domain Name:LOCALFOODCOOP.ORG
Domain ID: D152340160-LROR
Creation Date: 2008-04-10T18:03:18Z
Updated Date: 2014-03-28T21:30:18Z
Registry Expiry Date: 2015-04-10T18:03:18Z
Sponsoring Registrar:GoDaddy.com, LLC (R91-LROR)
Sponsoring Registrar IANA ID: 146
WHOIS Server: 
Referral URL: 
Domain Status: clientDeleteProhibited
Domain Status: clientRenewProhibited
Domain Status: clientTransferProhibited
Domain Status: clientUpdateProhibited
Registrant ID:CR35497609
Registrant Name:Robert Waldrop
Registrant Organization:Oklahoma Food Cooperative
Registrant Street: 1524 NW 21
Registrant City:Oklahoma City
Registrant State/Province:Oklahoma
Registrant Postal Code:73106
Registrant Country:US
Registrant Phone:+1.4056134688
Registrant Phone Ext: 
Registrant Fax: 
Registrant Fax Ext: 
Registrant Email:customer@oklahomafood.coop
...

**Comment by ayoungethan** [comment]
@guising is OKFC using OFS or an old LFC.org instance or...?  I wonder if someone would be amenable to forwarding the domain to OFS.org?  

All in all, agreed, it's not a priority.
